zeebe-workflow-manager/src/main/kotlin/workflowmanager/job/MoneyLoan.kt

```diff
@@ -1,23 +1,34 @@
 package workflowmanager.job
 
+import io.camunda.zeebe.client.ZeebeClient
 import io.camunda.zeebe.client.api.response.ActivatedJob
 import io.camunda.zeebe.client.api.worker.JobClient
 import io.camunda.zeebe.spring.client.annotation.JobWorker
+import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.stereotype.Component
 
 @Component
 public class MoneyLoan {
+    @Autowired
+    lateinit var zeebeClient: ZeebeClient
+
     @JobWorker(type = "send-acceptance-letter")
     fun sendAcceptanceLetter(client: JobClient, job: ActivatedJob) {
         client.newCompleteCommand(job.key).send().join()
         println("The loan request is accepted. Processing to transfer money.")
         Thread.sleep(1000)
     }
 
+    private var iteration = 0
     @JobWorker(type = "send-rejection-letter")
     fun sendRejectionLetter(client: JobClient, job: ActivatedJob) {
-        client.newCompleteCommand(job.key).send().join()
-        println("Sorry, your debt is too high to proceed. Loan request is rejected.")
+        client.newCompleteCommand(job.key).send()
+        if (iteration < 5) println("Sorry, your debt is too high to proceed. The rejection letter is sent. Waiting until the letter is received.")
+        iteration += 1
+        if (iteration == 5) {
```

Comment:
Looks like this will only ever evaluate as true once. Could you rather do it with modulo so every 5th request will evaluate to true? 

Like
```
if (iteration % 5 != 0)
```
and
```
if (iteration % 5 == 0)
```